web/lang/en_gb.php

```diff
@@ -421,6 +421,7 @@
     'MustConfirmPassword'   => 'You must confirm the password',
     'MustSupplyPassword'    => 'You must supply a password',
     'MustSupplyUsername'    => 'You must supply a username',
+    'NetworkScan'           => 'Scan Network',
```

Comment:
This is uneccessary.  The source text should be left as Scan Network. There is nothing wrong with spaces.